### ISSUE TITLE
Replace `aria-required` with `required` in ControlRange

### ISSUE THREAD

**Issue by katydecorah** [mapbox/mr-ui#117]
In another repository, the Lighthouse accessibility audit is failing on ControlRange due to:

> [aria-*] attributes do not match their roles

I ran Lighthouse accessibility audit on the test cases /ControlRange and saw the same result:

>![image](https://user-images.githubusercontent.com/2180540/109872007-d4cb3780-7c39-11eb-8df1-1a4787361dc6.png)
> ![image](https://user-images.githubusercontent.com/2180540/109872049-e3195380-7c39-11eb-9e37-2b69d3cf77d3.png)

## Solution

Replace `aria-required` with `required` in the ControlRange. 

From what I've learned, [we can safely move](https://caniuse.com/mdn-api_htmlinputelement_required) to using `required`:

> Note: HTML also specifies the required attribute which is supported well in user agents. Use aria-required for backwards compatibility only.
> &mdash; <https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-required_attribute>

Tagging @mapbox/studio as other known users of ControlRange, please let me know if you disagree with this suggested change.

**Comment by tristen** [comment]
No longer relevant after migrating to radix range primitives
